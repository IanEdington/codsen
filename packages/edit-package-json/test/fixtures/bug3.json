{"a": [],"gh": {"mn": "1","yz": "2"}}
