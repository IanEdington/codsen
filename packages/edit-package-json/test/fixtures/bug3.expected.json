{"a": [],"gh": {"mn": "1","yz": "3"}}
